configs/app.dev.json

```diff
@@ -0,0 +1,19 @@
+{
+    "type": "gpii.appWithTaskTray",
```

Comment:
As this config is based off the other config `app.json`. It can be simplified to:
```
{
    "type": "gpii.appWithTaskTray.dev",
    "options": {
        "gradeNames": ["fluid.component"],
            "distributeMenuDisplay": {
                "record": "gpii.app.menuInAppDev",
                "target": "{that menu}.options.gradeNames"
            }
        }
    },
    "mergeConfigs": [
        "./app.json"
     ]
}
```